ExampleMod/Content/Items/Mounts/ExampleMinecart.cs

```diff
@@ -0,0 +1,27 @@
+using ExampleMod.Content.Mounts;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Content.Items.Mounts
+{
+	public class ExampleMinecart : ModItem
+	{
+		public override void SetDefaults() {
+			Item.DefaultToMount(ModContent.MountType<ExampleMinecartMount>());
```

Comment:
Is there really any point calling this if you're going to override `width` and `height` anyway? Just set `mountType = ` directly

Reply:
I would wait until 1.4.5 vanilla has decided what to do with the method

Reply:
Please change to `mountType = `